docs/how-to/create-beamline.rst

```diff
@@ -68,12 +81,17 @@ The following example creates a fictitious beamline ``w41``, with a simulated tw
             bl_prefix=False,
         )
 
-    def d11(name: str = "D11") -> AdAravisDetector:
-        """
-        Also a valid Device factory function, but as multiple calls would instantiate
-        multiple copies of a device, discouraged.
-        """
-        return AdAravisDetector(name=name, prefix=f"{BL}-DI-DCAM-01:")
+    """
+    A valid factory function which:
+    - may be instantiated automatically
+        - cannot distinguish between simulated beamline and live
+        - each call instantiates a new copy
+    - must be explicitly connected (may be automated by tools)
+        - when connected may connect to a simulated backend
+        - may be connected concurrently (when automated by tools)
+    """
+    def synchrotron() -> Synchrotron:
+        return Synchrotron()
```

Comment:
Should: Why do we want this in the docs? Presumably we would like to encourage people to always be using `@device_factory`?